{"name": "test_rate_limit_error_handling", "status": "passed", "description": "Test handling of 429 rate limit errors", "attachments": [{"name": "log", "source": "021435a2-402d-4f4e-a0c4-cb96bc0ecfc2-attachment.txt", "type": "text/plain"}], "start": 1757711160271, "stop": 1757711160272, "uuid": "32f89cf8-046c-4d32-94ab-0cc6413f6304", "historyId": "dd5ada2a2050113c57f087410aefb46f", "testCaseId": "dd5ada2a2050113c57f087410aefb46f", "fullName": "tests.unit.test_llm_adapters_comprehensive.TestOpenAIAdapter#test_rate_limit_error_handling", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "parentSuite", "value": "tests.unit"}, {"name": "suite", "value": "test_llm_adapters_comprehensive"}, {"name": "subSuite", "value": "TestOpenAIAdapter"}, {"name": "host", "value": "Joshuas-MacBook-Pro-2.local"}, {"name": "thread", "value": "155-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.unit.test_llm_adapters_comprehensive"}], "titlePath": ["tests", "unit", "test_llm_adapters_comprehensive.py", "TestOpenAIAdapter"]}